039-AKSForEnterprise/Coach/README.md

```diff
@@ -0,0 +1,37 @@
+# What The Hack - AKS Integration with Azure Services
+
+## Introduction
+
+The main goal of this What The Hack is for participants who are already familiar with Kubernetes how to interact with the rest of the Azure platform.
+
+## Notes
+
+* Let participants find their own solutions, even if they are wrong. Let them hit walls and learn from their mistakes, unless you see them investing too much time and effort. Give them hints that put them on the right track, but not solutions
+* Most challenges can be solved in multiple ways, all of them correct solutions
+* If there is any concept not clear for everybody, try to make participants explain to each other. Intervene only when no participant has the knowledge
+* **Make sure no one is left behind**
+* Make sure participants have a way to share code, ideally git-based
+* Most challenges involve some level of subscription ownership to create identities or service principals, or for the AAD integration challenge.
+* Leave participants try options even if you know it is not going to work, let them explore on themselves. Stop them only if they consume too much time
+* For each challenge, you can ask the least participative members to describe what has been done and why
+
+> :warning: Note that the code snippets provided here are just an orientation for you as a coach, and might not work 100% in your particular environment. They have been tested, but the rapid nature of Azure CLI versions, Kubernetes, AKS, helm, etc makes it very difficult constantly reviewing them on a regular basis. If you find errors in the code, please send a PR to this repo with the correction.

```

Comment:
Please change this to match the style used in other What The Hacks. We don't use quoted sections. Just use a separate paragraph, as you are doing and prefix with a bolded "NOTE:" prefix, eg:

**NOTE:** The code snippets provided here are just an orientation for you as a coach, and might not work 100% in your particular environment. They have been tested, but the rapid nature of Azure CLI versions, Kubernetes, AKS, helm, etc makes it very difficult constantly reviewing them on a regular basis. If you find errors in the code, please send a PR to this repo with the correction.

Reply:
Done!